### PR TITLE
Remove missing method from host job template

### PR DESCRIPTION
**Pull request by lzap**
This template does not even exist, I don't know how I ended up adding it, but it does not work.

### CODE REVIEW

File: app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb [hunk 1]
```diff
@@ -18,4 +18,3 @@ model: WebhookTemplate
 # Task ended at <%= @object.task.ended_at %>
 # Task resulted with <%= @object.task.result %>
 # Task state <%= @object.task.state %>
-# Task action output <%= @object.task.action_output %>
```
Comment (ofedoren) [inline app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb:21]: I've found why it is not available anymore: https://github.com/theforeman/foreman-tasks/commit/b904dcfad3878365177e644cf8c8caf7b05cf0f5#diff-faa1c0e41ac8cbfd311a3e099f6da63d947a60f226b8227ccc6b3637601971c2

We need to rename it. Also, note that there are more templates in the repo that use `action_output`.

**Review by ofedoren — CHANGES_REQUESTED** [review]

File: app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb (hunk 1 above)
Comment (ofedoren) [reply app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb:21]: Also, if we want to not remove it then it worth to document that method as well (in foreman-tasks plugin)

Comment (lzap) [reply app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb:21]: Good catch, I think I will close this PR and create an alias in foreman-tasks.

**Comment by lzap** [comment]
@adamruzicka https://github.com/theforeman/foreman-tasks/pull/632

Alternatively, I can change all templates to the new method, backport it and make another release of webhooks.

**Comment by ofedoren** [comment]
@lzap, @adamruzicka confirmed that we need to rename the method and also remove it from Katello related templates.

**Comment by lzap** [comment]
Rename in tasks or in template? :-)

**Comment by adamruzicka** [comment]
Rename it in the template. The polls name was misleading and is only useful for a small subset of tasks

**Comment by lzap** [comment]
Rebased, l think we need to bump major version to 2.0 because this will be incompatible?

**Comment by ofedoren** [comment]
@lzap, well I'd treat this as a minor update/bug fix to be aligned with the tasks plugin. I don't see us introducing breaking changes here. Although it's hard to tell since we are not depending on the plugins explicitly, unfortunatelly.

**Comment by ofedoren** [comment]
@lzap, don't we want to remove the `action_output` from those:
 1: https://github.com/theforeman/foreman_webhooks/blob/ea5f7e07a8ddd7ef8a437f159252c03436295e2e/app/views/foreman_webhooks/webhook_templates/katello_-_promote.erb#L23
 2: https://github.com/theforeman/foreman_webhooks/blob/ea5f7e07a8ddd7ef8a437f159252c03436295e2e/app/views/foreman_webhooks/webhook_templates/katello_-_publish.erb#L24
 3: https://github.com/theforeman/foreman_webhooks/blob/ea5f7e07a8ddd7ef8a437f159252c03436295e2e/app/views/foreman_webhooks/webhook_templates/katello_-_repo_sync.erb#L27
 
 As far as I remember, @adamruzicka told that those are not even accessible for Katello actions.

**Comment by adamruzicka** [comment]
While it might be callable `@object.task.action_continuous_output` might be callable on all tasks, it will return something meaningful only for `Actions::RemoteExecution::RunHostJob`. For all others it will just return `nil`. Do we still want to mention it in the template even though it is pretty useless in most cases?

**Comment by lzap** [comment]
> Do we still want to mention it in the template even though it is pretty useless in most cases?

Absolutely not, sorry I did not understand what I was doing. I did not dig enough into tasks/rex/dynflow yet. So removing that should be the best option from what I understand now?

**Comment by adamruzicka** [comment]
I would say so. The method is there to allow generating reports about REX jobs, but is useless outside of that scope.

**Comment by lzap** [comment]
Removed the line.

**Review by adamruzicka — APPROVED** [review]

**Comment by lzap** [comment]
Pushing gem to https://rubygems.org...
You have enabled multi-factor authentication. Please enter OTP code.
Code:   XXXXXX
Successfully registered gem: foreman_webhooks (2.0.0)

**Pull request merged**